exercises/largest-series-product/example.cr

```diff
@@ -1,14 +1,13 @@
 class Series
+  @series : Array(Int64)
```

Comment:
It would seem so... I can't put it in the initializer and I can't seem to omit it. I'm not a Crystal master so for now I think this is a good fix. Might come up with something better but... until then!

Thanks!